src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelTranscript.tsx

```diff
@@ -96,7 +120,7 @@ const TrackPanelTranscript = (props: Props) => {
   return (
     <SimpleTrackPanelItemLayout
       visibilityStatus={trackVisibilityStatus}
-      onChangeVisibility={() => props.onChangeVisibility(trackVisibilityStatus)}
+      onChangeVisibility={() => onChangeVisibility()}
```

Comment:
`() => onChangeVisibility()` is identical to just `onChangeVisibility`

Reply:
✅